pkg/printers/internalversion/printers.go

```diff
@@ -762,27 +762,25 @@ func printPod(pod *api.Pod, options printers.GenerateOptions) ([]metav1.TableRow
 	for i := range pod.Status.InitContainerStatuses {
 		container := pod.Status.InitContainerStatuses[i]
 		restarts += int(container.RestartCount)
-		switch {
-		case container.State.Terminated != nil && container.State.Terminated.ExitCode == 0:
+
+		if container.State.Terminated != nil && container.State.Terminated.ExitCode == 0 {
```

Comment:
wait... I don't think this piece of code change is better.

How about this,

```go
switch {
case container.State.Terminated != nil:
    if container.State.Terminated.ExitCode == 0 {continue}
    // initialization is failed
    ....

case xxx:
    ....

default:
    ....
}
```

Reply:
Yes, I think it's good